onnx2kerastl/reshape_layers.py

```diff
@@ -323,11 +326,11 @@ def convert_flatten(node, params, layers, lambda_func, node_name, keras_name):
     logger.debug('Convert inputs to Keras/TF layers if needed.')
     input_0 = ensure_tf_type(layers[node.input[0]], name="%s_const" % keras_name)
 
-    # Fix critical issue with flatten
-    permute = keras.layers.Permute((3, 1, 2))
-    tensor_chw = permute(input_0)
-    flatten = keras.layers.Flatten(name=keras_name)
-    layers[node_name] = flatten(tensor_chw)
+    input_dims = input_0.shape
+    flatten_axis = params['axis']
```

Comment:
I think sometimes axis will not be shown so we can use the default value
```suggestion
    flatten_axis = params.get('axis', 1)
```